src/Utdanning/Seksjon.elm

```diff
@@ -206,7 +212,8 @@ nåværendeInfoTilUtdanningsSkjema nåværendeInfo =
 
 
 type Msg
-    = BrukerVilRegistrereUtdanning
+    = BrukerVilRegistrereUtdanning NivåValg
```

Comment:
Jeg syns kanskje ikke `NivåValg` hører til i Msgen her. Kan du ikke bare sjekke hvor brukeren skal sendes basert på aktivSamtale i `update` i stedet for å ha denne logikken i viewet?